### PR TITLE
Multiple tables

### PR DESCRIPTION
**Pull request by ashmalzahra**
- Create a table named `owners` with the following columns:
  - `id`: integer (set it as autoincremented PRIMARY KEY)
  - `full_name`: string
  - `age`: integer
- Create a table named `species` with the following columns:
  - `id`: integer (set it as autoincremented PRIMARY KEY)
  - `name`: string
- Modify `animals` table:
  - Make sure that id is set as autoincremented PRIMARY KEY
  - Remove column `species`
  - Add column `species_id` which is a foreign key referencing `species` table
  - Add column `owner_id` which is a foreign key referencing the `owners` table
- Remember all this goes in `schema.sql` file.
- Insert the following data into the `owners` table:
  - Sam Smith 34 years old.
  - Jennifer Orwell 19 years old.
  - Bob 45 years old.
  - Melody Pond 77 years old.
  - Dean Winchester 14 years old.
  - Jodie Whittaker 38 years old.
- Insert the following data into the `species` table:
  - Pokemon
  - Digimon
- Modify your inserted animals so it includes the `species_id` value:
  - If the name ends in "mon" it will be Digimon
  - All other animals are Pokemon
- Modify your inserted animals to include owner information (`owner_id`):
  - Sam Smith owns Agumon.
  - Jennifer Orwell owns Gabumon and Pikachu.
  - Bob owns Devimon and Plantmon.
  - Melody Pond owns Charmander, Squirtle, and Blossom.
  - Dean Winchester owns Angemon and Boarmon.
- Remember these insertions and modifications happen in `data.sql` file.
- Write queries (using `JOIN`) to answer the following questions:
  - What animals belong to Melody Pond?
  - List of all animals that are pokemon (their type is Pokemon).
  - List all owners and their animals, remember to include those that don't own any animal.
  - How many animals are there per species?
  - List all Digimon owned by Jennifer Orwell.
  - List all animals owned by Dean Winchester that haven't tried to escape.
  - Who owns the most animals?
- Remember all these should be written in `queries.sql` file.
q1
q2
q3

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -15,3 +15,39 @@ INSERT INTO animals ( name, date_of_birth, weight_kg, neutered, escape_attempts)
 INSERT INTO animals ( name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Blossom', '1998-10-13', 17, TRUE, 3);
 INSERT INTO animals ( name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Ditto', '2022-05-14', 22, TRUE, 4);
 
+/* Insert new table data. */
+
```
Comment (codecaiine) [inline data.sql:19]: - [x] Good job so far but You missed inserting the following data into the owner's table: **Sam Smith 34 years old** 😢  Please make sure to insert that information as required for this project 👍

File: data.sql [hunk 2]
```diff
@@ -15,3 +15,39 @@ INSERT INTO animals ( name, date_of_birth, weight_kg, neutered, escape_attempts)
...
+INSERT INTO owners ( full_name, age) VALUES ('Bob', 45);
+INSERT INTO owners ( full_name, age) VALUES ('Melody Pond', 77);
+INSERT INTO owners ( full_name, age) VALUES ('Dean Winchester', 14);
+INSERT INTO owners ( full_name, age) VALUES ('Jodie Whittaker', 38);
+
+INSERT INTO species (name) VALUES ('Pokemon');
+INSERT INTO species (name) VALUES ('Digimon');
+
+/* Modify existing data. */
+
```
Comment (codecaiine) [inline data.sql:31]: - [x] According to the project requirements you need to insert **animals to include owner information (owner_id): Sam Smith owns Agumon.** Please fix it 👍

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @ashmalzahra , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Good database and query setup ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED ✔️ 🎉 ✔️ 

Hi @ashmalzahra, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] Correct Gitflow ✔️ 
- [ ] Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**